src/nvim/CMakeLists.txt

```diff
@@ -912,6 +912,7 @@ file(GLOB API_SOURCES CONFIGURE_DEPENDS ${PROJECT_SOURCE_DIR}/src/nvim/api/*.c)
 
 file(GLOB LUA_SOURCES CONFIGURE_DEPENDS
   ${NVIM_RUNTIME_DIR}/lua/vim/*.lua
+  ${NVIM_RUNTIME_DIR}/lua/vim/_meta/*.lua
```

Comment:
was it an oversight that we didn't have this before?

Reply:
I think so. `make doc` was incorrectly saying the target is up-to-date.